sdgym/benchmark.py

```diff
@@ -76,16 +80,22 @@ def _synthesize(synthesizer_dict, real_data, metadata):
         data = list(real_data.values())[0]
         num_samples = len(data)
 
+    tracemalloc.start()
     now = datetime.utcnow()
     synthesizer_obj = get_synthesizer(data, metadata)
+    synthesizer_size = sys.getsizeof(synthesizer_obj) / (1024 * 1024)
```

Comment:
very minor: maybe we can pull 1024 * 1024 into a constant since it's used in different places in case we ever change the display units